client/my-sites/email/gsuite-purchase-cta/test/index.js

```diff
@@ -2,59 +2,33 @@
 /**
  * External dependencies
  */
-import { Provider } from 'react-redux';
+import { noop } from 'lodash';
 import React from 'react';
 import renderer from 'react-test-renderer';
 
 /**
  * Internal dependencies
  */
-import { createReduxStore } from 'state';
-import GSuitePurchaseCta from '../';
 
+import { GSuitePurchaseCta } from '../';
```

Comment:
why does this need to be deconstructed?

Reply:
This is the un-connected Component being imported so I can test so it can be tested without a test Redux store.

Reply:
ahhh, like it